src/StudioCore/Assets/Paramdex/ER/Meta/BulletParam.xml

```diff
@@ -6,6 +6,8 @@
         <Option Value="1" Name="Strike" />
         <Option Value="2" Name="Thrust" />
         <Option Value="3" Name="Standard" />
+        <Option Value="253" Name="Refers to EquipParamWeapon atkAttribute2 field" />
+        <Option Value="254" Name="Refers to EquipParamWeapon atkAttribute field" />
```

Comment:
bruh if it wasn't for fromsoft declaring this unsigned, I'd say sign it - this is clearly -2 and -3 what are they playing at